Source/PeleLM_derive.cpp

```diff
@@ -165,7 +189,7 @@ void pelelm_dercpmix (const Box& bx, FArrayBox& derfab, int dcomp, int /*ncomp*/
         }
         Temp = in_dat(i,j,k,1);
         EOS::TY2Cp(Temp,Yt,Cpmix);
-        der(i,j,k) = Cpmix * 1.0e-4; // CGS -> MKS ;
+        der(i,j,k+dcomp) = Cpmix * 1.0e-4; // CGS -> MKS ;
```

Comment:
it should be k,dcomp ?

Reply:
Yeah, but it's better done in the different style.  See new version.